lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -40,6 +47,44 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     }
   }
 
+  function handleKeyDown(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (event.isDefaultPrevented()) {
+      return;
+    }
+
+    const tabElements = event.currentTarget.parentElement?.children;
+
+    const getNextElementFromEventKey = (key: string) => {
+      switch (key) {
+        case 'Left':
+        case 'ArrowLeft':
+          return tabElements?.[index - 1] ?? tabElements?.[tabElements.length - 1];
+        case 'Right':
+        case 'ArrowRight':
+          return tabElements?.[index + 1] ?? tabElements?.[0];
+        case 'Home':
+          return tabElements?.[0];
+        case 'End':
+          return tabElements?.[tabElements.length - 1];
+        default:
+          return;
+      }
+    };
+
+    const nextElement = getNextElementFromEventKey(event.key);
+
+    if (nextElement) {
+      event.preventDefault();
+      (nextElement as HTMLElement).focus();
+    }
+  }
+
+  function handleFocus() {
+    if (activationMode === 'automatic' && activeTab !== index) {
```

Comment:
It would be easier to read and slightly more efficient if an `isActiveTab` local variable were extracted out for use here and in the `tabIndex` computation below